### PR TITLE
Travis to docker.

### PR DESCRIPTION
**Pull request by luca-heltai**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,77 +1,19 @@
...
-  - cd build
-  - cmake -GNinja -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DENABLE_ALL_EXECUTABLES=ON ..
-  - ninja -j2
-  - ctest -j2 -V -R "${BUILD_TYPE,,}"
-
-
-branches:
-  only:
-    - master
+- docker run -P -v `pwd`:/home/dealii/pi-DoMUS:rw heltai/deal2lkit:$BUILD_TYPE /bin/sh -c "mkdir pi-DoMUS/build; cd pi-DoMUS/build; cmake -GNinja -DENABLE_DOCUMENTATION=OFF  -DENABLE_ALL_EXECUTABLES=ON ../; ninja"
```
Comment (asartori86) [inline .travis.yml]: I would put ninja -j4, since 6 might be too much for the small ram we have

File: .travis.yml [hunk 2]
```diff
@@ -1,77 +1,19 @@
...
+- docker run -P -v `pwd`:/home/dealii/pi-DoMUS:rw heltai/deal2lkit:$BUILD_TYPE /bin/sh -c "cd pi-DoMUS/build; ctest  -V"
```
Comment (asartori86) [inline .travis.yml]: ctest -j2?

**Comment by asartori86** [comment]
apart from the small suggestions looks great! thanks!

File: .travis.yml (hunk 1 above)
Comment (luca-heltai) [inline .travis.yml]: if you don't specify anything, ninja uses all available processors by default. Do we know in advance how many are assigned to our travis boxes?

Comment (asartori86) [inline .travis.yml]: if nothing is specified ninja sets to 6.  according to https://docs.travis-ci.com/user/ci-environment/ we have ~2 cores, bursted. But we have more ram than the container (the previous environment) indeed we have 7.5GB

**Pull request merged**